state-chain/runtime/src/chainflip/bitcoin_elections.rs

```diff
@@ -111,19 +111,14 @@ impls! {
 	}
 
 	/// Associating the state machine and consensus mechanism to the struct
-	StateMachineES {
+	StatemachineElectoralSystemTypes {
 		// both context and return have to be vectors, these are the item types
 		type OnFinalizeContextItem = ();
 		type OnFinalizeReturnItem = ChainProgress<btc::BlockNumber>;
 
-		// restating types since we have to prove that they have the correct bounds
-		type Consensus2 = InputHeaders<Self>;
-		type Vote2 = InputHeaders<Self>;
-		type VoteStorage2 = vote_storage::bitmap::Bitmap<InputHeaders<Self>>;
-
 		// the actual state machine and consensus mechanisms of this ES
 		type ConsensusMechanism = BlockHeightTrackingConsensus<Self>;
-		type StateMachine = BlockHeightTrackingSM<Self>;
+		type Statemachine = BlockHeightTrackingSM<Self>;
```

Comment:
We still have `BwStateMachine` with capital M. (Don't mind which one we use, but yeah, should be consistent)

Reply:
also `StateMachineES` and the assoc type `StateMachine`

Reply:
Yeah thanks, should be all now.